server/services/booking.js

```diff
@@ -49,6 +49,14 @@ const bookingServiceFactory = (eliteApi, keyworkerApi, allocationManagerApi) =>
     }
   }
 
+  const isOffenderRecordRetained = async (context, offenderNo) => {
+    try {
+      return dataComplianceApi.isOffenderRecordRetained(context, offenderNo)
+    } catch (error) {
+      return false
```

Comment:
log error? but wondering if it's good to be handling errors both here and inside the gateway.

Reply:
Do you need an await in here? Similarly to L46 above: 
`return await keyworkerApi.getKeyworkerByCaseloadAndOffenderNo(context, activeCaseLoadId, offenderNo)`

Reply:
Error logged now, await added (although don't think the await is necessary since the consuming code was handling it either way)

Reply:
If you don't `await` the catch will never be called.